webpack/webpack/webpack.dev.babel.js

```diff
@@ -36,6 +33,7 @@ module.exports = require('./webpack.base.babel')({
   // Add hot reloading
   plugins: [
     new webpack.HotModuleReplacementPlugin(), // Tell webpack we want hot reloading
+    new webpack.NoErrorsPlugin(),
```

Comment:
What does the `NoErrorsPlugin` do and why do we need it? Don't we want to see errors?